app/controllers/rooms_controller.rb

```diff
@@ -5,6 +5,13 @@ def show
     @furnitures = @room.furnitures
   end
 
+  def search
+    @rooms = Room.eager_load(:furnitures).search_furnitures(params[:key])
```

Comment:
`.eager_load(:furnitures).search_furnitures(params[:key])` gộp thành 1 scope thôi là dc á
```suggestion
    @rooms = Room.search_name_furnitures(params[:key])
```

`scope :search_name_furnitures, ->(key){joins(:furnitures).where "furnitures.name LIKE ?", "%#{key}%"}`